Sources/MapboxCoreNavigation/Router.swift

```diff
@@ -13,9 +13,9 @@ public protocol RouterDataSource: AnyObject {
 }
 
 /**
- A route and its index in a `RouteResponse` that sorts routes from most optimal to least optimal.
+ A `RouteResponse` object that sorts routes from most optimal to least optimal and selected route index in it.
  */
-public typealias IndexedRoute = (Route, Int)
+public typealias IndexedRouteResponse = (routeResponse: RouteResponse, routeIndex: Int)
```

Comment:
@mapbox/navigation-ios  Does anybody know the reason for using type alias here instead of a full featured type like struct?

Reply:
I think it is a good opportunity to refactor this type in a struct unless there is a strong reason not to.

Reply:
Done